web/releases/tenzir-platform-v1.10/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Tenzir Platform v1.10: A New Explorer"
+slug: tenzir-platform-v1.10
+authors: [lava]
+date: 2025-05-15
+tags: [release, platform]
+comments: true
+---
+
+Today, we're releasing [Tenzir Platform v1.10][github-release], featuring
+a revamped Nodes page and the option to statically define a set of workspaces
+in an on-prem environment.
+
+![Tenzir Platform v1.10](tenzir-platform-v1.10.svg)
+
+[github-release]: https://github.com/tenzir/platform/releases/tag/v1.10.0
+
+<!-- truncate -->
+
+## The Nodes Page
+
+[...]
+
+## Static Workspaces and Ephemeral Nodes
+
+Users of the Sovereign Edition of the Tenzir Platform now have the option to
+
+```yaml
+---
+workspaces:
+  static0:
+    name: Tenzir
+    category: Statically Configured Workspaces
+    icon_url: https://storage.googleapis.com/tenzir-public-data/icons/tenzir-logo-square.svg
+    auth_rules:
+      - {"auth_fn": "auth_allow_all"}
+```
+
+To conveniently obtain the auth rules in the correct format, use the
+new `print-auth-rule` command of the CLI. For example, to get the rule
+above you can run `tenzir-platform tools print-auth-rule allow-all`.
+
+To find the static workspace configuration file, the Tenzir Platform container
+looks at the `WORKSPACE_CONFIG_FILE` environment variable.
+
+### Ephemeral Nodes
+
+For statically configured workspaces, it is possible to define a workspace
+token. The workspace token is a shared secret, that allows every Tenzir Node
+in possession of the token to connect to this workspace.
+
+```yaml
+workspaces:
+  static0:
+    token: wsk_e9ee76d4faf4b213745dd5c99a9be11f501d7009ded63f2d5NmDS38vXR
+```
+
+Please note that a valid workspace token must follow a specific format.
+To obtain a suitable token for your workspace, use
+the `tenzir-platform tools generate-workspace-token` command.
+
+Instead of writing out the workspace token in plain text, it is also possible
+to specify a file that contains the token:
+
+```yaml
+workspaces:
+  static0:
+    token_file: /run/secrets/workspace_token
```

Comment:
In the node's config file we generally use dashes over underscores in keys, so this likely should be adapted.